### PR TITLE
[WIP] Update full-icu and chromedriver

### PR DESCRIPTION
**Pull request by dervos**
Update chromedriver and full-icu to support yarn.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
     "babel-preset-react": "~6.24.1",
     "babel-preset-stage-0": "~6.24.1",
     "babel-register": "~6.24.1",
-    "chromedriver": "~2.26.1",
+    "chromedriver": "^2.30.1",
```
Comment (djhi) [inline package.json]: Please use tilde

File: package.json [hunk 2]
```diff
@@ -43,7 +43,7 @@
     "express": "~4.15.2",
     "extract-text-webpack-plugin": "~1.0.1",
     "file-api": "~0.10.4",
-    "full-icu": "~1.0.3",
+    "full-icu": "^1.1.3",
```
Comment (djhi) [inline package.json]: Same

**Comment by fzaninotto** [comment]
e2e tests fail

**Comment by dervos** [comment]
Yes, I noticed. I couldn't reproduce the failing e2e test on my machine, those ran fine here. I could use some directions as to why these tests seem to fail on travis.

**Comment by fzaninotto** [comment]
>  I could use some directions as to why these tests seem to fail on travis.

Sorry, no time for that. If you think you can't troubleshoot that yourself, then please close the PR.

Bu the way, can you explain why you think this PR is necessary?

**Comment by dervos** [comment]
This version of `full-icu` failed to install while using yarn. This rendered me and perhaps other people unable to use yarn for `admin-on-rest`. `full-icu@1.1.3` would've fixed it.

**Comment by fzaninotto** [comment]
But then why did you update chrome driver and selenium?

**Comment by dervos** [comment]
You're right. Upgrading those too could cause the issues. I've made a new PR here: #897.

**Pull request closed without merging**